Applications/Spire/Source/KeyBindings/CancelKeyBindingsModel.cpp

```diff
@@ -0,0 +1,51 @@
+#include "Spire/KeyBindings/CancelKeyBindingsModel.hpp"
+#include "Spire/Spire/ValidatedValueModel.hpp"
+
+using namespace boost;
+using namespace Spire;
+
+CancelKeyBindingsModel::CancelKeyBindingsModel() {
+  for(auto i = 0; i < 13; ++i) {
```

Comment:
Do not use magic numbers:

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-magic